package.json

```diff
@@ -7,18 +7,18 @@
     "type": "git",
     "url": "https://github.com/DEFRA/ffc-demo-web.git"
   },
-  "main": "index.js",
+  "main": "app/index.js",
   "scripts": {
     "build": "npm run build:css",
-    "build:css": "node-sass --output-style=expanded --output=server/public/build/stylesheets client/sass",
-    "build:css-watch": "node-sass --output-style=expanded --output=server/public/build/stylesheets --watch client/sass",
+    "build:css": "node-sass --output-style=expanded --output=app/static/stylesheets app/static/stylesheets",
+    "build:css-watch": "node-sass --output-style=expanded --output=app/static/stylesheets --source-map true --watch app/static/stylesheets",
     "pretest": "npm run test:lint",
     "test": "npm run test:unit",
     "test:lint": "standard",
     "test:unit": "jest",
     "unit:watch": "jest --coverage=false --onlyChanged --watch",
```

Comment:
When developing locally I make a lot of use of 'test watch' when working on new features or making changes. It would be good to supplement the test watch script with docker-compose settings to call it. We don't necessarily need to add one now, but I'd be likely to add one as soon as I start making code changes to a project.